sw_firewall/README.md

```diff
@@ -1,14 +1,27 @@
 # Software Firewall PoC
 
-The Proof of Concept is made up of three types of containers: traffic sources,
-traffic targets, and proxies.  The initial PoC will be:
+The Proof of Concept is made up of three types of containers:
+
+- traffic sources
+- traffic targets
+- proxies
+
+The initial PoC will be:
+
+- IPDK P4-eBPF container for networking
+- Docker-based container setup using Docker Compose
+- Server containers hostring traffic sources
+- Traffic target containers
+- Proxy containers
+
+## Second PoC
+
+The Second Proof of Concept is made up of three types of containers: traffic
+sources, traffic targets, and proxies.  This second PoC will be:
 - Servers hosting traffic sources sending traffic to
 - Servers with IPU or DPU cards, with traffic targets on the servers
 - Proxies running on IPU or DPU cards
 
```

Comment:
Is this really a second PoC? Or is it a PoC on top of the initial p4-ebpf one ? 
Don't both of these together provide the FireWall PoC?

Reply:
The main separation is that they are in fact two different PoCs, and we may not use p4-ebpf on top of IPU/DPU for the second PoC.

Reply:
It's not clear in the flow of the README, there is a description of first poc and second poc, then running the ipdk PoC
Perhaps moving the "Running the PoC" Before elaborating on the second PoC will clarify that ? 
Or separate files per PoC ?

Reply:
sure